modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -154,4 +159,85 @@ spec:
     secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581 <1>
 ...
 ----
-<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
\ No newline at end of file
+<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
+
+[id="workload-resource-mapping_{context}"]
+== Workload resource mapping
+
+[NOTE]
+====
+* Workload resource mapping is available for the secondary workloads of the ServiceBinding custom resource (CR) for both the API groups: `binding.operators.coreos.com` and `servicebinding.io`.
+* You must namespace `ClusterWorkloadResourceMapping` resources only under the `servicebinding.io` API group. However, the `ClusterWorkloadResourceMapping` resources interact with `ServiceBinding` resources under both the `binding.operators.coreos.com` and `servicebinding.io` API groups.
+====
+
+The specification also provides a way to define exactly where binding data needs to be projected. Use this method when you are not able to configure custom path locations correctly by previous methods. You can define `ClusterWorkloadResourceMapping` resources only in the `servicebinding.io` API group, which specify where binding data is to be projected for a given workload kind.
+
+.Example: Schema of the `ClusterWorkloadResourceMapping` resource
+[source,yaml]
+----
+apiVersion: servicebinding.io/v1alpha3
+kind: ClusterWorkloadResourceMapping
+metadata:
+  name: # string <1>
+  ...
+spec:
+  versions: # required, at least one entry must be defined
+  - version: # string <2>
+    annotations: # string (Fixed JSONPath), optional <3>
+    containers: # optional
+    - path: # string (JSONPath) <4>
+      name: # string (Fixed JSONPath), optional <5>
+      env: # string (Fixed JSONPath), optional <6>
+      volumeMounts: # string (Fixed JSONPath), optional <7>
+    volumes: # string (Fixed JSONPath), optional <8>
```

Comment:
In examples, please use example values or `<replaceable strings in brackets>`. Avoid comments like this.

Reply:
Replaceable values should follow this format: `<ip_address>`

Reply:
I don't see another schema in the openshift-docs repo that uses comments this way. Please use replacement values and callouts in the typical fashion instead.

Reply:
We avoid repeating because over time, one set of information might get updated while the other doesn't. This produces confusing or conflicting information and maintenance headaches.

Reply:
Discussed this with @Srivaralakshmi.  We've replaced this with a more minimal example.